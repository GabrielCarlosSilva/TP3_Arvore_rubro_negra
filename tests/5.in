1
7
Marlene
42
Mariana
29
Luana
20
Vitoria
25
Liana
38
Elisa
16
Marcos
18
2
1
3
Mauricio 
70
Lucas
23
Lauro 
48
2
0
