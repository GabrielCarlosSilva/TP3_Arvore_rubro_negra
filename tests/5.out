Dados inOrder:
Nome: Elisa
Idade: 16
Nome: Marcos
Idade: 18
Nome: Luana
Idade: 20
Nome: Vitoria
Idade: 25
Nome: Mariana
Idade: 29
Nome: Liana
Idade: 38
Nome: Marlene
Idade: 42
Dados inOrder:
Nome: Elisa
Idade: 16
Nome: Marcos
Idade: 18
Nome: Luana
Idade: 20
Nome: Lucas
Idade: 23
Nome: Vitoria
Idade: 25
Nome: Mariana
Idade: 29
Nome: Liana
Idade: 38
Nome: Marlene
Idade: 42
Nome: Lauro
Idade: 48
Nome: Mauricio
Idade: 70